### PR TITLE
remake

### PR DESCRIPTION
**Pull request by jonathanarde**
remake export playlist feature with simpler logic,
showing default language checkbox

### CODE REVIEW

File: public/javascripts/playlistClicked.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 var playlistClickedComponent = (function(){
-    var songs = m.prop(playlist.songs)
-    var playlistName = m.prop(playlist.name)
-
+    var songs = m.prop(playlist_songs)
```
Comment (chrislim) [inline public/javascripts/playlistClicked.js]: where are the playlist_songs and playlist_name variable being defined now?

File: routes/user.playlist.js [hunk 2]
```diff
@@ -25,58 +27,60 @@ router.get('/', function(req, res, next) {
...
-                    .populate('songs')
-                    .exec(function(err, afterDelete) {
-                        res.send({
-                            msg: 'deleting done',
-                            songs: afterDelete.songs
-                        })
-                    })
-            })
-        })
+// router.post('/', function(req, res, next) {
```
Comment (chrislim) [inline routes/user.playlist.js]: still including these methods?

File: views/export3.jade [hunk 3]
```diff
@@ -3,34 +3,35 @@ extends layout
 block content
     include topnav
     h1 Preview
-    each songs in songs2d
+    //Note: songs2d = [{songs: [Song Schema that the user checked]}, {minLine: the smallest lyric line}]    
+    each songsObj in songs2d
         -var i = 0
         -var j = 0
-        -while (i < songs.length - 1 && j < songs.length - 1)
+        -while (i < songsObj.songs.length && j < songsObj.songs.length)
```
Comment (chrislim) [inline views/export3.jade]: so was the former length -1 a bug or are we dealing with something different now?

Comment (jonathanarde) [inline views/export3.jade]: before i put the minLine on the last index of array like [[obj, obj, minLine], [obj, minLine]], now i change it to be [{song: [obj], minLine: Integer}]

Comment (chrislim) [inline views/export3.jade]: Ah i see, ok.

**Pull request merged**